src/tty/unix.rs

```diff
@@ -206,6 +228,10 @@ impl Pty {
             die!("ioctl TIOCSWINSZ failed: {}", errno());
         }
     }
+
+    pub fn child_info(&self) -> ChildInfo {
+        ChildInfo { pid: self.child.id() as _ }
+    }
```

Comment:
If we turn `trait EventedReadWrite` into `trait Pty` or similar, this should either go there, or maybe come back in a tuple from `Pty::new`.